model/ravelWrap.cc

```diff
@@ -494,7 +494,8 @@ namespace minsky
         else
           throw error(ravel_lastErr());
       }
-    v.hypercube({}); // ensure scalar data space allocated
+      // This line causes an empty result from ravel output port
+    //v.hypercube({}); // ensure scalar data space allocated
```

Comment:
This is still needed, but only if the hypercube setting on line 488 hasn't taken place. Perhaps best handled by adding a return statement on line 493.

Reply:
Okay, made these changes. Unfortunately did them in separate commits. Sorry.